AcloConcurrency/AcloConcurrency/Networking/Endpoints/FilterEndpoints.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 enum FilterEndpoints {
-    case ingredient(Ingredient)
+    case ingredient(name: String)
```

Comment:
ingredient(String)

Reply:
Why you do that? If you wan't add more ingredients, use Enum Ingredient. It help us don't make a spelling mistake

Reply:
do u know all ingredients for Enum? U fetch this data from fck api))))